extendr-api/src/robj/mod.rs

```diff
@@ -802,7 +800,7 @@ make_typed_slice!(Rint, INTEGER, INTSXP);
 make_typed_slice!(f64, REAL, REALSXP);
 make_typed_slice!(Rfloat, REAL, REALSXP);
 make_typed_slice!(u8, RAW, RAWSXP);
-make_typed_slice!(Rstr, STRING_PTR, STRSXP);
```

Comment:
See line here. STRING_PTR is removed from the public API and it _looks like_ that STRING_PTR_R0 and other `_R0` might be user facing now? 

Is this your read as well?
https://github.com/r-devel/r-svn/blob/48e02881502dce9eeeefff1bc422bc0968913c04/src/include/Defn.h#L427

Reply:
Yes. They made functions that return const-ptr instead of mutable-ptr, so this makes sense. The `VECTOR_PTR` version has been deprecated for a while now.